content/source/docs/enterprise/install/installer.html.md

```diff
@@ -182,6 +185,33 @@ ADD example-intermediate-ca.crt /usr/local/share/ca-certificates/
 RUN update-ca-certificates
 ```
 
+#### RHEL 7
+
+Note that in addition to the packages installed by yum, the `daemontools` packaged is required as it provides the `envdir` binary,
+which is used by TFE. If this package is not installed on the alternative worker image, Terraform plan and apply operations will fail.
+
+```
+FROM registry.access.redhat.com/rhel7
+
+# running 'subscription-manager' is necessary to be able to install additional packages via yum
+# substitute your redhat account credentials in this command
+RUN subscription-manager register --username ${RH_username} --password "${RH_password}" --auto-attach
+
+# Include all necessary CA certificates.
+ADD example-root-ca.crt /usr/share/pki/ca-trust-source/anchors
+ADD example-intermediate-ca.crt /usr/share/pki/ca-trust-source/anchors
+
+# Update the CA certificates bundle to include newly added CA certificates.
+RUN update-ca-trust
+
+# Install software used by Terraform Enterprise.
```

Comment:
The passive voice bothered me, so I tried rewording :-) Can you change the one in Ubuntu as well?

```suggestion
# Install required software for Terraform Enterprise.
```

Reply:
No problem! Done :+1: